File: src/main/java/com/github/weewar/mapviewer/exceptions/ImagePreloadException.java
package com.github.weewar.mapviewer.exceptions;

public class ImagePreloadException extends RuntimeException {
    public ImagePreloadException() {
        super();
    }

    public ImagePreloadException(String message) {
        super(message);
    }

    public ImagePreloadException(String message, Throwable cause) {
        super(message, cause);
    }

    public ImagePreloadException(Throwable cause) {
        super(cause);
    }
}

File: src/main/kotlin/com/github/weewar/mapviewer/exceptions/ImagePreloadException.kt
package com.github.weewar.mapviewer.exceptions

class ImagePreloadException : RuntimeException {
    constructor() : super() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable) : super(cause) {}
}
